packages/cli/src/lib/auth.ts

```diff
@@ -59,7 +64,67 @@ export async function login(root: string, shop?: string | true) {
     ]);
   });
 
+  await hideLoginInfo();
+
   const config = await setShop(root, session.storeFqdn);
 
   return {session, config};
 }
+
+function showLoginInfo() {
+  let deferredResolve: (value?: unknown) => void;
+  const promise = new Promise((resolve) => {
+    deferredResolve = resolve;
+  });
+
+  console.log('');
+
+  let hasLoggedTimeout = false;
+  let hasLoggedPressKey = false;
+
+  const restoreLogs = muteAuthLogs({
+    onKeyTimeout: (url) => {
+      if (url) {
+        hasLoggedTimeout = true;
+        process.stdout.write(ansiEscapes.eraseLines(9));
+        renderInfo({
+          headline: 'Log in to Shopify',
+          body: [
+            `\nTimed out. Click to open your browser:`,
+            {link: {url, label: colors.white('Open')}},
+          ],
+        });
+      }
+    },
+    onPressKey: () => {
+      hasLoggedPressKey = true;
+      renderInfo({
+        headline: 'Log in to Shopify',
+        body: 'Press any key to login with your default browser',
```

Comment:
I thought about that but it might be a bit intrusive to do it automatically?
cc @mynameisadamf Should we do this and change the text to simply "Opening your default browser..." or similar?

As mentioned in OP, unfortunately we can't show the link to open the browser from the beginning yet.

Reply:
My opinion is that it shouldn't do it automatically.